File: codigos/1 - estrutura sequencial/1005 - Media 1/Main.java
import java.io.IOException;
import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static String getMedia(double n1, double n2){
        double media = (n1*3.5 + n2*7.5)/11;
        return String.format("%.5f", media);
    }
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner prompt = new Scanner(System.in);
        System.out.println("MEDIA = " + getMedia(prompt.nextDouble(), prompt.nextDouble()));
    }
}

File: codigos/1 - estrutura sequencial/1005 - Media 1/Main.clj
(defn calcular-media [A B]
  (/ (+ (* 3.5 A) (* 7.5 B)) 11.0))

(defn main []
  (let [A (read)
        B (read)
        media (calcular-media A B)]
      (println "MEDIA =" (format "%.5f" media))))
  
(main)